src/TerminalObject/Helper/Art.php

```diff
@@ -87,7 +87,7 @@ protected function artDir($art)
      */
     protected function artFile($art)
     {
-        $files = $this->fileSearch($art, '[^' . \DIRECTORY_SEPARATOR . ']*$');
+        $files = $this->fileSearch($art, '[^' . '\\' . DIRECTORY_SEPARATOR . ']*$');
```

Comment:
It's important to prevent `DIRECTORY_SEPARATOR` from being overriden by specifying the global namespace: `\DIRECTORY_SEPARATOR`.

Maybe useless concatenation could be avoided:
`'[^\\' . \DIRECTORY_SEPARATOR . ']*$'`

Reply:
Actually, I think it's preferable to use preg_quote rather than always adding a backslash: `/` does not need to be escaped.